### PR TITLE
Feature setup

### PR DESCRIPTION
**Pull request by kkolade**
- Project set up using Gitflow workflow
- Create an npm project with webpack.
- Write HTML markup with minimum styling 
- Use provided wireframe as a guide
- No API calls made so far 
- Use ES6 modules, with import and export.

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @kkolade 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that the relevant linters were not configured. Kindly configure the linters and ensure that they pass 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/template.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            <form action="" class="add-score-form">
+              <input type="text" name="" id="" placeholder="Your name" />
+              <input type="text" name="" id="" placeholder="Your Score" />
+              <br />
+              <button class="submit">Submit</button>
+            </form>
+          </div>
+        </main>
+      </div>
+    </div>
```
Comment (omarsalem7) [inline src/template.html]: - [ ] Kindly make your HTML code looks like in the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png)
- [ ] I got the following error when I run your project, kindly fix this issue
>image

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @kkolade 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- The Pull Request has a proper title and description. ✔️
- Correct git-flow ✔️

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kkolade** [comment]
@Gambit142 thanks for your review but I must disagree with your submissions:
- There is no one way to write HTML the important thing is the look in the browser, your submission is a personal opinion and not necessarily HTML best practice, my code was written semantically
- The error is because of dependencies on your machine, which I have no control over, please read the error message carefully and follow the suggestion in the message and the error will disappear

**Comment by kkolade** [comment]
@omarsalem7 thanks for your review but I must disagree with your submissions:

- There is no one way to write HTML the important thing is the look in the browser, your submission is a personal opinion and not necessarily HTML best practice, my code was written semantically
- The error is because of dependencies on your machine, which I have no control over, please read the error message carefully and follow the suggestion in the message and the error will disappear

**Comment by kkolade** [comment]
@omarsalem7 Please I will appreciate your response, the error message says to run 'npm rebuild node-sass' to clear the error message. I need to be ready for the next stage of the project.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @kkolade, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ All linters passing
✔️ Error on run fixed

 ## Required Changes ♻️
- [x] Kindly make sure to follow the wireframe as suggested by the previous reviewer.
Screenshot 2023-01-11 at 2 07 04 AM

## Optional suggestions

- [x] As it was not pointed out by the previous reviewer this is marked optional but it is an important requirement to add a dist folder. Have a look at the directory.

Screenshot 2023-01-11 at 2 02 39 AM

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Abenezer-Tilahun — APPROVED** [review]
Hi @kkolade 
 
Your project is complete  👍 💯   There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 👏🏻

✔️ No linter errors.
✔️ You used correct Git-Flow.
✔️ Professional README  
✔️ Good Commit Messages
✔️ Good job for Implemented Suggested changes.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**